docs/content/usage/configuration/integration.md

```diff
@@ -0,0 +1,24 @@
+---
+title: "Integration"
+date: 2023-11-28T10:08:52Z
+draft: false
+description: "Configure Infection Monkey by import/export a configuration file."
+---
+
+## Export Configuration
+
+You can export the current configuration with or without a password using **Export** button. It will save any changes to the configuration to a file which you can later import.
```

Comment:
A little bit more detail is needed regarding what exporting with/without a password does and why someone might like to choose one over the other.